R/MsBackend.R

```diff
@@ -182,10 +182,14 @@
 #' @param value replacement value for `<-` methods. See individual
 #'     method description or expected data type.
 #'
-#' @param values for `filterValues()`: A `numeric` vector that define the
+#' @param values For `filterValues()`: A `numeric` vector that define the
 #'     values to filter the `object`. `values` needs to be of same length than
 #'     parameter `spectraVariables` and in the same order.
 #'
+#' @param y For `cbind2()`: A `data.frame` or `DataFrame` with the
+#'     spectra variables to be added to the backend. Need to be of the same
```

Comment:
I would maybe be a bit more precise here:

*The number of rows and their order has to match the number of spectra in `x`, respectively their order.* instead of *Need to be ...*. Or something along these lines, but mention that a) the number of rows has to match AND that the order is important.